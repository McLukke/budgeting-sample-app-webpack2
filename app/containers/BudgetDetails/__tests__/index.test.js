import React from 'react';
import renderer from 'react-test-renderer';
import { BudgetDetails } from '../';

// mock nested component
jest.mock('components/DonutChart');

it('renders correctly', () => {
  const mockTransaction = {
    catgeoryId: "6",
    description: "Gas",
    id: 2,
    value: -764.73
  };
  const mockTransactions = [mockTransaction];

  const tree = renderer.create(
    <BudgetDetails
      transactions={mockTransactions}
      selectedTransaction={mockTransaction}
    />
  ).toJSON();
  expect(tree).toMatchSnapshot();
});
