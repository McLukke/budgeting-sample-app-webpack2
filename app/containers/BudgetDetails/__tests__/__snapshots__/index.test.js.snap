// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div>
  <h1>
    Gas
  </h1>
  <h2
    className="neg"
  >
    -
    100
    %
  </h2>
  <button
    onClick={[Function]}
  >
    Go Back
  </button>
</div>
`;
